Sources/JWTKit/Utilities/BigNumber.swift

```diff
@@ -27,12 +27,12 @@ class BigNumber {
         return BigNumber(c)
     }
 
-    public func toBase64(_ size: Int = 1000) -> String {
+    public func toBase64URL(_ size: Int = 1000) -> String {
```

Comment:
We'll need to keep this around since it's a public function (to avoid breaking the public API). Given there are other instances when a Base 64 string might be useful I suggest we just create a new function with the correct behaviour